tests/test_pinky.rs

```diff
@@ -68,5 +58,8 @@ fn test_short_format() {
 
 #[cfg(target_os = "linux")]
 fn expected_result(args: &[&str]) -> String {
-    TestScenario::new(UTIL_NAME).cmd(UTIL_NAME).args(args).run().stdout
+    use std::process::Command;
+
+    let output = Command::new(UTIL_NAME).args(args).output().unwrap();
+    String::from_utf8_lossy(&output.stdout).into_owned()
```

Comment:
ah yeah. .ucmd() runs the uutils pinky, .cmd(UTIL_NAME) will run the pinky on the host